src/registrar/models/domain_request.py

```diff
@@ -1182,6 +1166,10 @@ def _is_special_district_complete(self):
         # Special District -> "Election office" and "About your organization" page can't be empty
         return self.is_election_board is not None and self.about_your_organization is not None
 
+    # Do we still want to test this after creator is autogenerated? Currently it went back to being selectable
```

Comment:
I believe Erin left this comment, I wonder if it's maybe bc of a flag that is not turned on? I went through the domain request flow and I don't see the creator in the request bc of the profile feature flag (I think that's what she means by autogenerated maybe?) and it was not selectable in the domain request flow

I'll give her the floor to respond though bc I don't want to speak for her in case I'm way off, but that's just my assumption 😅